spec/features/dams_collections_spec.rb

```diff
@@ -302,4 +302,45 @@
       visit osf_api_dams_collection_path @provCollection2.pid
       expect(page).to have_content('{"name":"UC San Diego Library"}')
     end
-end
\ No newline at end of file
+end
+
+feature 'Visitor wants to look at the collection item view search results' do
+  before do
+    @unit = DamsUnit.create(pid: 'bb45454545')
+    @unit.damsMetadata.content = File.new('spec/fixtures/damsUnit.rdf.xml').read
+    @unit.save!    
+    @aCollection = DamsAssembledCollection.create(pid: "xx4473712z", visibility: "public")
+    @aCollection.damsMetadata.content = File.new('spec/fixtures/soccomCollection.rdf.xml').read
+    @aCollection.save!
+    @soccomObj1 = DamsObject.create(pid: "xx2801340v")
+    @soccomObj1.damsMetadata.content = File.new('spec/fixtures/soccomObject1.rdf.xml').read
+    @soccomObj1.save!
+    @soccomObj2 = DamsObject.create(pid: "xx47126209")
+    @soccomObj2.damsMetadata.content = File.new('spec/fixtures/soccomObject2.rdf.xml').read
+    @soccomObj2.save!
+    @soccomObj3 = DamsObject.create(pid: "xx66239018")
+    @soccomObj3.damsMetadata.content = File.new('spec/fixtures/soccomObject3.rdf.xml').read
+    @soccomObj3.save!
+    solr_index (@unit.pid)
+    solr_index (@aCollection.pid)
+    solr_index (@soccomObj1.pid)
+    solr_index (@soccomObj2.pid)   
+    solr_index (@soccomObj3.pid)       
+  end
+  after do
+    @aCollection.delete
+    @unit.delete
+    @soccomObj1.delete
+    @soccomObj2.delete
+    @soccomObj3.delete
+  end 
+  scenario 'should see the collection item result page with correct indentation' do
```

Comment:
Given the change to strip the html tags, I'm wondering if that's what this test should be more focused on. It's less about correct indentation (the side-effect) and more about assuring that any fields that we'd display as highlights in the search results don't contain html tags, is that right? I'm not sure if there's a more efficient way to test that specifically, off the top of my head, but maybe we should rename the scenario line?